maps/src/G3SkyMap.cxx

```diff
@@ -965,6 +965,11 @@ PYBINDINGS("maps") {
 	       "Compute the sky coordinates, expressed as quaternion rotations "
 	       "from the pole, for each of the given 1-D pixel coordinates.")
 
+	    .def("query_disc", &G3SkyMap::QueryDisc,
+	       (bp::arg("alpha"), bp::arg("delta"), bp::arg("radius")),
+	       "Return a list of pixel indices whose centers are located "
+	       "a disc of the given radius at the given sky coordinates.")
+
```

Comment:
I'm pretty sure all the return and argument types here are caught by the numpy_vector_infrastructure() stuff in core/src/python.cxx, but it might be worth checking that explicitly. You could use a printf() or just see if numpy.asarray(retval)[3] = 5 actually sets retval[3] to 5. If they *aren't* caught, this will be really slow.

Reply:
Also, there is a missing word in the docstring: "are located [within] a disc"

Reply:
Great! Can you fix the typo?